File: 0075-sort-colors/0075-sort-colors.py
from typing import List

class Solution:
    def sortColors(self, nums: List[int]) -> None:
        no_of_zeroes = 0
        no_of_ones = 0
        
 
        for i in range(len(nums)):
            if nums[i] == 0:
                no_of_zeroes += 1
            elif nums[i] == 1:
                no_of_ones += 1
        
      
        for i in range(len(nums)):
            if i < no_of_zeroes:
                nums[i] = 0
            elif i < no_of_zeroes + no_of_ones:
                nums[i] = 1
            else:
                nums[i] = 2

File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] arr) {
        int n = arr.length;
        int noofzeroes=0,noofone=0;
        for(int i=0;i<n;i++){
            if(arr[i]==0) noofzeroes++;
            if(arr[i]==1) noofone++;
        }
         for(int i=0;i<n;i++){
            if(i<noofzeroes) arr[i]=0;
            else if(i<noofzeroes+noofone) arr[i]=1;
            else arr[i]=2;
        }
       
        
        
    }
}
